packages/main/src/themes/RatingIndicator.css

```diff
@@ -14,39 +15,74 @@
 	cursor: initial;
 }
 
-:host([readonly]) .ui5-rating-indicator-icon.ui5-rating-indicator-readonly-unselected-icon {
+:host([readonly]) .ui5-rating-indicator-readonly-item-unsel {
 	color: var(--sapContent_UnratedColor);
 }
 
+:host(:not([readonly]):not([disabled])) .ui5-rating-indicator-root:hover {
+	opacity: .9;
+}
+
 :host([_focused]) {
 	outline: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
 	border-radius: var(--_ui5_rating_indicator_border_radius);
 }
 
 .ui5-rating-indicator-root {
 	outline: none;
+	position: relative;
+}
+
+.ui5-rating-indicator-list {
+	list-style-type: none;
+	display: flex;
+	margin: 0;
+	padding: 0;
 }
 
-.ui5-rating-indicator-icon {
+.ui5-rating-indicator-item {
 	position: relative;
-	color: var(--sapContent_UnratedColor);
-	user-select: none;
+	width: var(--_ui5_rating_indicator_item_width);
+	height: var(--_ui5_rating_indicator_item_height);
+	min-width: var(--_ui5_rating_indicator_item_width);
+    min-height: var(--_ui5_rating_indicator_item_height);
 }
 
-.ui5-rating-indicator-icon.ui5-rating-indicator-active-icon {
-	color: var(--sapContent_RatedColor);
+.ui5-rating-indicator-item:not(:last-child) {
+	margin-right: 0.1875rem;
```

Comment:
You would probably need to mirror this `margin-right` in RTL as `margin-left`? 
If yes, please use `margin-inline-end` , instead of `margin-right` and the browser will automatically add the margin at the right or at the left, based on the page direction.

Reply:
Done.